cockroachcloud/managing-cmek.md

```diff
@@ -236,11 +236,76 @@ To rotate the CMEK keys for one or more cluster regions:
       --data "@cmek_config.json"
     ```
 
+## Add a region to a multi-region CMEK-enabled cluster
```

Comment:
I don't like this heading, because it's not yet a multi-region cluster until you add a second region (or in another way, every single-region cluster is just a multi-region cluster with only one region so far). Why not just "to a CMEK-enabled cluster"?

Reply:
fair point.